scalalib/src/CrossModuleBase.scala

```diff
@@ -6,9 +6,8 @@ import mill.define.Cross.Resolver
 import mill.eval.PathRef
 
 object CrossModuleBase {
-  def scalaVersionPaths(scalaVersion: String, f: String => os.Path) = {
```

Comment:
Should I maintain the old function as deprecated for backwards binary compatibility?

Reply:
I think we also break binary compatibility by the other changes too, but am not sure if that is an issue with current external plugins. I tend to keep this PR open a bit longer and merge before next major bump, if that is OK with you?

Reply:
It's totally fine either way. This particular question is not a problem anymore since you suggested to leave the `CrossScalaModule` untouched and leave only the utility methods + infrastructure for it to work instead of changing the default behaviour.

Reply:
Fine. So we wait for next major (0.10.0) release. I hope we get some MiMa integration into it, too.

That said, if there are others who think we should integrate your initial change to `CrossScalaModule`, please speak up!

Reply:
> Should I maintain the old function as deprecated for backwards binary compatibility?

Sorry, for my late change of mind, but your initial question directed me to the question of binary compatibility, but this `def` is probably also used in users code, so yes, now I think we should still keep it around and `@deprecate` it with a good hint.